File: Python/Data Structures & Algorithms Module - Python/Week1/PersonApp.py
"""
PersonApp.py
Billy Cussen
09/11/2020
"""
import Person
from Person import myPerson
import Student
from Student import myStudent

person1 = myPerson("Billy","Cussen",31)
person2 = myStudent("William","Cussen",32)

myList = []
myList.append(person1)
myList.append(person2)

for i in myList:
    if isinstance(i,myStudent):
        print(i)
        print(i.isStudent())

File: Java/Data Structures & Algorithms Module - Java/Week8/Revision/Week1/PersonApp.java
/*
PersonApp.java
Billy Cussen
09/11/2020
*/
import java.util.List;
import java.util.ArrayList;
public class PersonApp{

    public static void main(String [] args){

        Person one = new Person("Billy","Cussen",31);
        Person two = new Student("William","Cussen",32);

        List<Person> myList = new ArrayList<Person>();
        myList.add(one);
        myList.add(two);

        for (Person e: myList){
            if (e instanceof Student){
                Student s = (Student) e;
                System.out.println(s.toString());
                System.out.println(s.isStudent());
            }
        }
    }
}